### PR TITLE
TCM-118 | Build dummy welcome screen

### PR DESCRIPTION
**Pull request by halilyesilyurt**
### Summary

- [x] Completed dummy welcome screen.

### Issue

[TCM-118](https://hyfnla.atlassian.net/browse/TCM-118)

### Video/Screenshot

welcome

### CODE REVIEW

File: app/src/main/res/layout/fragment_welcome.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:textColor="@color/white"
+        android:textSize="28sp"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/lblDescription"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <com.trx.consumer.common.CommonLabel
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_welcome.xml:82]: Needs `android:fontFamily="@font/atcarquette_regular"` and `android:textSize="15sp"`

File: app/src/main/res/layout/fragment_welcome.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:text="@string/welcome_get_started_label"
+        android:textAllCaps="true"
+        android:textColor="@color/yellow"
+        android:textSize="12sp"
+        android:textStyle="bold"
```
Comment (VivekV95) [inline app/src/main/res/layout/fragment_welcome.xml:63]: Let's use `android:fontFamily="@font/atcarquette_bold"`

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Couple of small design changes

File: app/src/main/res/layout/fragment_welcome.xml (hunk 2 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_welcome.xml]: Completed.

File: app/src/main/res/layout/fragment_welcome.xml (hunk 1 above)
Comment (halilyesilyurt) [reply app/src/main/res/layout/fragment_welcome.xml:82]: Completed.

**Review by VivekV95 — APPROVED** [review]

**Review by onodude — DISMISSED** [review]
thanks!

**Review by onodude — APPROVED** [review]
thanks!

**Pull request merged**